### PR TITLE
add Discord link to navbar

### PR DESCRIPTION
**Pull request by andreynering**
Followup of https://github.com/go-gitea/website/pull/46

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -46,38 +46,7 @@ menu:
       url: https://github.com/go-gitea/
       weight: 60
       pre: github
-
```
Comment (lunny) [inline config.yaml]: Why remove these?

Comment (andreynering) [inline config.yaml]: @lunny Commented code. I think it shouldn'd be there? If we ever need that again we can copy from website or docs repo.

**Comment by lunny** [comment]
LGTM

**Comment by appleboy** [comment]
LGTM

**Pull request merged**